config/gaiatest_testvars.json

```diff
@@ -101,6 +101,13 @@
 		"device_cert": "cert_device",
 		"branch_cert": "v2.0",
 		"buildname_cert": "cert_build-.master.B-xxx.Gecko-xxxxxxx.Gaia-xxxxxxx"
+	},
+	"visualtest": {
+		"enabled": true,
+		"engine": "pil",
+		"save_baseline": false,
+		"baseline_dir": "",
+		"cmp_dir": ""
```

Comment:
Don't forget to update .boe and .mad files, so they are copied to their respective machines